internal/compiler/widgets/native/std-widgets.slint

```diff
@@ -257,3 +257,88 @@ export GridBox := GridLayout {
     spacing: NativeStyleMetrics.layout-spacing;
     padding: NativeStyleMetrics.layout-spacing;
 }
+
+export StandardTableView := Rectangle {
+    callback sort-ascending(int);
+    callback sort-descending(int);
+    
+    out property <int> current-sort-column: -1;
```

Comment:
should that be a in-out property?
Or should it just be private?

Reply:
I think it is an important information for the user but because there is no property changed callback and there is extra logic for sorting you have to use the `sort` function.